modules/4_pvs_support/pvs_support.tf

```diff
@@ -196,8 +196,13 @@ resource "null_resource" "adjust_mtu" {
   provisioner "remote-exec" {
     inline = [<<EOF
 export HTTPS_PROXY="http://${var.vpc_support_server_ip}:3128"
+if [ "$(oc get Network.config cluster -o jsonpath='{.status.networkType}')"  == "OpenShiftSDN" ]
+then
+oc patch Network.operator.openshift.io cluster --type=merge --patch   '{"spec": { "migration": { "mtu": { "machine": { "to" : 9100} } } } }'
```

Comment:
Let's keep this same if-then-logic
If the patches turn out to be the same, let's still leave it for ease of use.